requirements-base.txt

```diff
@@ -1,6 +1,9 @@
 BeautifulSoup>=3.2.1
 boto3>=1.4.1,<1.4.6
 botocore<1.5.71
+
+# sentry-plugins specific dependencies
```

Comment:
oof, the fix imports pre-commit hook really likes to do this, I'd just `commit -n` this